Terran-One/cw-simulate-ui
# Expand Sidebar by default + various bugfixes

## Issue link
- [WL-433](https://terran-one.atlassian.net/browse/WL-433)
- [WL-449](https://terran-one.atlassian.net/browse/WL-449)
- [WL-450](https://terran-one.atlassian.net/browse/WL-450)
- [WL-451](https://terran-one.atlassian.net/browse/WL-451)

## Description
- expand sidebar menus by default
- fix UI showing up for invalid or non-existent chain
- fix uninitialized CWSimulateEnv introducing countless errors
- fix errors relating to `undefined`
- fix premature `storeCode` call (WL-449)
- fix wrong default chain in new simulation (WL-450)
- fix missing code in new simulation (WL-451)
- fix "Add" button in ContractUploadModal being clickable without valid upload
- remove unused recoil selector

## Test steps
1. Create a new simulation on welcome screen.
2. Verify sidebar is fully expanded (specifically: Chains + Codes) & has the just uploaded code.
4. Upload a new code through Codes subtree.
5. Before clicking Add, verify that the chosen code is not already in sidebar.
6. Click Cancel. Verify that the chosen code still is not already in sidebar.
7. Upload code & Add. Verify that the chosen code is now indeed in sidebar.
8. Create a new chain. Verify that the newly created chain is fully expanded.
